jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MultiPart.java

```diff
@@ -1378,7 +1379,7 @@ private boolean parseContent(Content.Chunk chunk)
                 int position = buffer.position();
                 int length = boundaryOffset;
                 // BoundaryFinder is configured to search for '\n--Boundary';
-                // if we found '\r\n--Boundary' then the '\r' is not content.
+                // if '\r\n--Boundary' is found, then the '\r' is not content.
```

Comment:
the code above this block is:
```java
if (boundaryOffset == 0)
    crContent = false;
if (boundaryOffset >= 0)
{
    notifyCRContent();
```

This should be:

```java
if (boundaryOffset >= 0)
{
    if (boundaryOffset == 0)
        crContent = false;
    else
        notifyCRContent();
```

but with better comments

Reply:
Done.